packages/slate-editor/src/extensions/placeholders/components/PlaceholderElement.tsx

```diff
@@ -13,6 +13,7 @@ import { type Props as BaseProps, Placeholder } from './Placeholder';
 export type Props = RenderElementProps &
     Pick<BaseProps, 'icon' | 'title' | 'description' | 'format' | 'onClick' | 'onDrop'> & {
         element: PlaceholderNode;
+        removable: boolean;
```

Comment:
I wasn't sure whether to make this optional and make it default to `true` since it's already being provided by the extension configuration.